tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -836,6 +837,92 @@ def fully_connected(inputs,
 
 
 @add_arg_scope
+def layer_norm(inputs,
+               center=True,
+               scale=False,
+               epsilon=0.001,
+               activation_fn=None,
+               reuse=None,
```

Comment:
One of the features on layer normalization is that normalization happens across features/neurons in a layer and so each sample in a batch is independent. This means that the mean and variance is calculated the same during both training and inference and we don't need moving averages.